static/src/js/components/AuthCallbackContainer/AuthCallbackContainer.jsx

```diff
@@ -9,17 +11,22 @@ import { useSearchParams } from 'react-router-dom'
 
 export const AuthCallbackContainer = () => {
   const [searchParams] = useSearchParams()
+  const { updateLoginInfo, user } = useAppContext()
+
   const path = searchParams.get('target')
+  const auid = searchParams.get('auid')
 
   const navigate = useNavigate()
 
   useEffect(() => {
-    if (path) {
-      navigate(path)
+    updateLoginInfo(auid)
```

Comment:
Not a huge fan of the fact we need to pass the auid through to this function. I think the only way around it would be to move the router provider outside of the auth provider, which I am not a huge fan of either...

Reply:
Yeah agree, I even tried adding ;auid=${auid} to the cookie, so we wouldn't have to pass it via the header but there is something going on in AWS from sending multiple cookies to the browser.